### PR TITLE
Branch v1.4 manual testing refine

### LINKED ISSUE #240: Dereference introduction from table of contents
'Welcome  to TA<sup>2</sup>` should not be in table of contents.

### PR DESCRIPTION
**Pull request by Droffilc13**
Introduce more space to test case.

Correct ignore headings for toc tag.

Change email in test case to correct email being referred

fix #240

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -964,31 +967,39 @@ testers are expected to do more *exploratory* testing.
...
    
 4. Adding a person with missing compulsory fields.
+
       1. Prerequisites: Ensure that no person in your contact list has the name "Steve Jobs".
-      1. Some invalid formats of `add` command you can try are `add`, `add m/cs2100 n/Steve Jobs` and `add n/Steve Jobs`. <br/>
+   
+      2. Some invalid formats of `add` command you can try are `add`, `add m/cs2100 n/Steve Jobs` and `add n/Steve Jobs`. <br/>
       Expected: No person is added. Error details shown in the status message due to invalid command format.
    
 ### Deleting a person
```
Comment (qinguorui2001) [inline docs/DeveloperGuide.md:1001]: consistent spacing, under other titles, there 's no extra empty line

**Review by qinguorui2001 — CHANGES_REQUESTED** [review]
just change spacing

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Droffilc13) [reply docs/DeveloperGuide.md]: Good catch

**Review by qinguorui2001 — APPROVED** [review]
LGTM

**Pull request merged**